missing requirements for google-cloud-core?

The error:
```
ImportError while importing test module 'python-docs-samples/jobs/v3/api_client/auto_complete_sample_test.py'.                
Hint: make sure your test modules/packages have valid Python names.                                                                                               
Traceback:                                                                                                                                                        
auto_complete_sample_test.py:18: in <module>                                                                                                                      
    from gcp_devrel.testing import eventually_consistent                                                                                                          
.nox/py-3-6/lib/python3.6/site-packages/gcp_devrel/testing/eventually_consistent.py:18: in <module>                                                               
    from google.cloud import exceptions                                                                                                                           
E   ModuleNotFoundError: No module named 'google.cloud' 
```

--- a/setup.py
+++ b/setup.py
@@ -21,6 +21,7 @@ with io.open('README.rst', 'r') as fh:
     long_description = fh.read()
 
 requirements = [
+    'google-cloud-core',
     'requests',
     'retrying',
     'setuptools >= 25.0.0',

